src/common/TrackedOp.h

```diff
@@ -121,6 +121,7 @@ class OpTracker {
    * @return True if there are any Ops to warn on, false otherwise.
    */
   bool check_ops_in_flight(std::vector<string> &warning_strings);
+  bool check_ops_in_flight(std::vector<string> &warning_strings, int &slow);
```

Comment:
I think the style guide only allows const references anyway; output parameters should all be pointers.